storybookjs/storybook
# UI: Remove scrolling attribute from iFrame

Issue: #11799

The iframe that renders the stories has a scrolling attribute that is both unnecessary and deprecated.

## What I did
Remove the deprecated scrolling attribute from the iframe.


## How to test

- Is this testable with Jest or Chromatic screenshots? One could test the iframe for the absence of the attribute, but this would not make much sense.
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.
